src/main/java/org/jabref/gui/entryeditor/OtherFieldsTab.java

```diff
@@ -72,8 +74,9 @@ protected Set<Field> determineFieldsToShow(BibEntry entry) {
         Optional<BibEntryType> entryType = entryTypesManager.enrich(entry.getType(), mode);
         if (entryType.isPresent()) {
             Set<Field> allKnownFields = entryType.get().getAllFields();
-            Set<Field> otherFields = entry.getFields().stream().filter(field -> !allKnownFields.contains(field)).collect(Collectors.toCollection(LinkedHashSet::new));
-
+            Set<Field> otherFields = entry.getFields().stream()
+                                          .filter(field -> !allKnownFields.contains(field) && !field.getName().toLowerCase().contains("comment"))
```

Comment:
It should be `startsWith` and `comment-`, because [UserSpecificCommentField.java](https://github.com/JabRef/jabref/pull/9727/files#diff-36401c049949316b318933b778a9613bfbfd565775e41b28ddcd2ddb286a44cd) is implemented like that.

Maybe also check for plain `comment` in addition.

Reply:
Why does the following not work here?

(that code is from above)

```java
   field -> field.equals(StandardField.COMMENT) || field instanceof UserSpecificCommentField
```